### ISSUE TITLE
Error connecting to placement service

### ISSUE THREAD

**Issue by stather** [dapr/dapr#859]
## In what area(s)?

/area placement

## What version of Dapr?

> 0.2.x

## Actual Behavior

`== DAPR == time="2019-11-28T22:09:18+02:00" level=debug msg="error establishing client to placement service: rpc error: code = Unavailable desc = all SubConns are in TransientFailure, latest connection error: connection error: desc = \"transport: Error while dialing dial tcp [::1]:6050: connectex: No connection could be made because the target machine actively refused it.\"`

Always getting the error above when starting an actor with dotnetsdk. Any ideas why this would be happening?
    running in minikube
  
dapr 0.2.0

## Steps to Reproduce the Problem

Basically just using the sample actor from the tutorial/demo using the dotnet sdk
All the rest of the output looks fine, but never manages to connect to the placement service.

I have tried uninstalling dapr and reinstalling again, in a fresh instance of minikube. Get same result...

**Comment by yaron2** [comment]
I see port `6050` is used, which is what we use to connect from the Dapr CLI to the placement service running locally on Windows.

You mentioned running in Minikube, for which the Dapr CLI would not be used with. The Dapr CLI connects to the local placement service. If you are running in `kubernetes` mode, the Dapr containers in your pod will be connecting to the placement service running under the `dapr` namespace.

**Comment by stather** [comment]
Sorry, even though I am running minikube this test was using local cli as below...

PS C:\Users\russell.stather\github\dotnet-sdk\samples\Actor\DemoActor> dapr run --log-level debug --port 3500 --app-id demo_actor --app-port 5000 dotnet run
You're up and running! Both Dapr and your app logs will appear here.

```
== DAPR == time="2019-12-03T11:59:30+02:00" level=info msg="starting Dapr Runtime -- version 0.1.0 -- commit 4358565-dirty"
== DAPR == time="2019-12-03T11:59:30+02:00" level=info msg="log level set to: debug"
== DAPR == time="2019-12-03T11:59:30+02:00" level=info msg="standalone mode configured"
== DAPR == time="2019-12-03T11:59:30+02:00" level=info msg="loaded component statestore (state.redis)"
== DAPR == time="2019-12-03T11:59:30+02:00" level=info msg="application protocol: http. waiting on port 5000"
== APP == Hosting environment: Development
== APP == Content root path: C:\Users\russell.stather\github\dotnet-sdk\samples\Actor\DemoActor
== APP == Application started. Press Ctrl+C to shut down.
== DAPR == time="2019-12-03T11:59:32+02:00" level=info msg="application discovered on port 5000"
== APP == info: Microsoft.AspNetCore.Hosting.Diagnostics[1]
== APP ==       Request starting HTTP/1.1 GET http://127.0.0.1:5000/dapr/config application/json
== APP == info: Microsoft.AspNetCore.Hosting.Diagnostics[2]
== DAPR == 2019/12/03 11:59:32 redis: connecting to 172.17.199.181:6379
== DAPR == 2019/12/03 11:59:32 redis: connected to 172.17.199.181:6379 (localAddr: 172.17.199.177:56793, remAddr: 172.17.199.181:6379)
== DAPR == time="2019-12-03T11:59:42+02:00" level=warning msg="error initializing pub sub pubsub.redis: redis streams: error connecting to redis at localhost:6379: dial tcp [::1]:6379: connectex: No connection could be made because the target machine actively refused it."
== DAPR == time="2019-12-03T11:59:42+02:00" level=info msg="actor runtime started. actor idle timeout: 1h0m0s. actor scan interval: 30s"
== DAPR == time="2019-12-03T11:59:42+02:00" level=info msg="actors: starting connection attempt to placement service at localhost:6050"
== DAPR == time="2019-12-03T11:59:42+02:00" level=info msg="http server is running on port 3500"
== DAPR == time="2019-12-03T11:59:42+02:00" level=info msg="gRPC server is running on port 56780"
== DAPR == time="2019-12-03T11:59:42+02:00" level=info msg="local service entry announced"
== DAPR == time="2019-12-03T11:59:42+02:00" level=info msg="dapr initialized. Status: Running. Init Elapsed 11306.203ms"
== DAPR == time="2019-12-03T11:59:44+02:00" level=debug msg="error establishing client to placement service: rpc error: code = Unavailable desc = all SubConns are in TransientFailure, latest connection error: connection error: desc = \"transport: Error while dialing dial tcp [::1]:6050: connectex: No connection could be made because the target machine actively refused it.\""
== DAPR == time="2019-12-03T11:59:46+02:00" level=debug msg="error establishing client to placement service: rpc error: code = Unavailable desc = all SubConns are in TransientFailure, latest connection error: connection error: desc = \"transport: Error while dialing dial tcp [::1]:6050: connectex: No connection could be made because the target machine actively refused it.\""
== DAPR == time="2019-12-03T11:59:49+02:00" level=debug msg="error establishing client to placement service: rpc error: code = Unavailable desc = all SubConns are in TransientFailure, latest connection error: connection error: desc = \"transport: Error while dialing dial tcp [::1]:6050: connectex: No connection could be made because the target machine actively refused it.\""
== DAPR == time="2019-12-03T11:59:52+02:00" level=debug msg="error establishing client to placement service: rpc error: code = Unavailable desc = all SubConns are in TransientFailure, latest connection error: connection error: desc = \"transport: Error while dialing dial tcp [::1]:6050: connectex: No connection could be made because the target machine actively refused it.\""
== DAPR == time="2019-12-03T11:59:54+02:00" level=debug msg="error establishing client to placement service: rpc error: code = Unavailable desc = all SubConns are in TransientFailure, latest connection error: connection error: desc = \"transport: Error while dialing dial tcp [::1]:6050: connectex: No connection could be made because the target machine actively refused it.\""
== DAPR == time="2019-12-03T11:59:55+02:00" level=info msg="dapr shutting down. Waiting 5 seconds to finish outstanding operations"
== DAPR == time="2019-12-03T11:59:55+02:00" level=info msg="stop command issued. Shutting down all operations"
```

**Comment by stather** [comment]
ok fixed it. So the placement service is running in minikube and it has a mapping set up  as 
```
6640f076f40b        daprio/dapr                                                      "./placement"            6 minutes ago       Up 6 minutes        0.0.0.0:6050->50005/tcp
        dapr_placement
```
This means you have to use the --placement-host cli parameter to connect to the ip addressp provided by

`minikube docker-env
`
Thanks
